### ISSUE TITLE
i have an a idea to config.ini

### ISSUE THREAD

**Issue by Tmuru321** [Megabyte918/MultiOgar-Edited#1143]
It is possible to add a configuration in config.ini that what you do is that of the percentage that one needs to eat another player and also the absorption of how far one must be to be able to eat it, this idea came to me when I remember the nonexistent Ogar unlimited I had those options and I thought it would be great to add them to MultiOgar-edited but I do not know anything about programming :c
maybe its impossible idk.
I am the basic of programming so do not talk to me about technical things
If it is not possible it does not matter

**Comment by Tmuru321** [comment]
sorry for my english

**Comment by chewydev** [comment]
I see what you mean, it would be a good idea for this to be added, although its the developer's choice.

**Comment by AlexHGaming** [comment]
@chewydev I AM GETTING THE ALMIGHTY MEGABYTE

How to trigger him:

@Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 @Megabyte918 
christian

**Comment by Tmuru321** [comment]
That spam

**Comment by Tmuru321** [comment]
Wish he saw it

**Comment by AlexHGaming** [comment]
Me and him talk in dm's like nearly every day. He is doing something
"related" to it.

**Comment by Tmuru321** [comment]
👍

**Comment by Agarian** [comment]
Anybody else have the idea to change `gameserver.ini` to `config.ini`? Just for the sake of people who are new to the server?

**Comment by fayizan** [comment]
@Agarian how about ```gameconfig.ini``` ?

**Comment by Megabyte918** [comment]
Collision resolution already has to grab the `mobilePhysics` config and perform square root calculations to get the eating distance every time an entity eats another entity (food, viruses, ejected mass, etc) which is slow, so adding another config for the eat-size requirement would make it even slower. If you would like to change the value manually however, you can find it [here](https://github.com/Megabyte918/MultiOgar-Edited/blob/master/src/GameServer.js#L774). (It's 1.15)

TL;DR: Yes it's possible, but I won't be doing it for performance reasons. 

I've been working on converting m-ed to use spatial hashing rather than a quad-tree for collision detection, viewbox updating, etc. So hopefully that will make a lot more room for performance and fun stuff such as extra configs & gamemodes :)
